### PR TITLE
187277049-generate_missing_ce_assessments_for_dce

### PR DESCRIPTION
**Pull request by ttoomey**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description
based to 108 since I think that's what we are using for migration work

[PT issue](https://www.pivotaltracker.com/story/show/187277049)

## Type of change
[//]: # 'remove options that are not relevant'
- [x] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/synthetic_ce_assessments_for_custom_assessment.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        create_synthetic_ce_event(custom_assessment, default_attrs)
+        total += 1
+      end
+    end
+    total
+  end
+
+  protected
+
+  def create_synthetic_ce_event(custom_assessment, default_attrs)
```
Comment (gigxz) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/synthetic_ce_assessments_for_custom_assessment.rb]:
```suggestion
  def create_synthetic_ce_assessment(custom_assessment, default_attrs)
```

File: drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/synthetic_ce_assessments_for_custom_assessment.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      AssessmentLevel: assessment_level,
+      PrioritizationStatus: prioritization_status,
+    }
+    prepared = custom_assessments.preload(enrollment: :project).preload(:form_processor)
+    total = 0
+    Hmis::Hud::Assessment.transaction do
+      prepared.find_each do |custom_assessment|
+        next if custom_assessment.form_processor.ce_assessment
+
+        create_synthetic_ce_event(custom_assessment, default_attrs)
```
Comment (gigxz) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/synthetic_ce_assessments_for_custom_assessment.rb]:
```suggestion
        create_synthetic_ce_assessment(custom_assessment, default_attrs)
```

**Review by gigxz — APPROVED** [review]

**Comment by gigxz** [comment]
@eanders flagging this as release-108 since migration work is happening next week. Or, we can start working off a deploy branch. Let me know what you prefer!

**Comment by eanders** [comment]
@gigxz 108 should be fine at this point.

**Pull request merged**